### PR TITLE
MGDAPI-5313 initial commit

### PR DESCRIPTION
**Pull request by MStokluska**
Initial commit for version 0.1.0

### CODE REVIEW

File: on-prem-to-rhoam/README.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
+# On-prem to RHOAM migrations
+
+Based on the agreement between the Customer and Red Hat the migration steps owners will vary, however the general flow remains the same and it is a matter of who is going to go through each SOP.
```
Comment (MStokluska) [inline on-prem-to-rhoam/README.md]:
```suggestion
 The owner of each migration step in the process will vary based on the level of access granted by the customer, however, the general flow remains the same and it is a matter of who is going to go through each SOP.
```

File: on-prem-to-rhoam/README.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+# Migration Paths
+
+These are the available migration paths based on the level of permissions given to Red Hat
```
Comment (MStokluska) [inline on-prem-to-rhoam/README.md]: remove this or #9

File: README.md [hunk 3]
```diff
@@ -1 +1,8 @@
-# Managed-Api-Service-Migrations
\ No newline at end of file
+# Managed-Api-Service-Migrations
+
+This repository covers migrations from 3scale on-prem and SaaS to Managed Api Service (RHOAM)
```
Comment (laurafitzgerald) [inline README.md]:
```suggestion
This repository covers migrations from 3scale on-prem and SaaS to Red Hat OpenShift API Management (RHOAM)
```

**Review by carlkyrillos — COMMENTED** [review]
@MStokluska I think it may be easier for future readers if the folders under `on-prem-to-rhoam` were named rather than just `approach_1`, `approach_2`, etc.

For example:

- `approach_1` -> `some-permissions`
- `approach_2` -> `full-permissions`
- `approach_3` -> `limited-permissions`
- `approach_4` -> `no-permissions`


These are just suggestions though, there may be better names. Other than that I think this looks good.

**Pull request merged**